docs/docs/examples/agent/coa_agent.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip llama-index-packs-agents-coa
</pre><p>did you mean pip install </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>